sdk/core/azure-core/azure/core/settings.py

```diff
@@ -114,6 +115,28 @@ def convert_logging(value: Union[str, int]) -> int:
     return level
 
 
+def convert_azure_cloud(value: Union[str, AzureClouds]) -> AzureClouds:
```

Comment:
Do we need this to be public?

Reply:
Other converters are all public. I don't think we need to treat it specially.